source/administration/changelog.md

```diff
@@ -149,18 +149,21 @@ The following deprecations are planned for the Mattermost v6.0 release, which is
 
 7. [Experimental Timezone](https://docs.mattermost.com/administration/config-settings.html#timezone) configuration setting.
 
-8. [All configuration settings previously marked as “Deprecated”](https://docs.mattermost.com/administration/config-settings.html#deprecated-configuration-settings).
-
-9. Legacy Sidebar experimental configuration settings, including:
-   - "EnableLegacySidebar"
-   - "ExperimentalTownSquareIsReadOnly"
-   - "ExperimentalHideTownSquareinLHS"
-   - "EnableXToLeaveChannelsFromLHS"
-   - "CloseUnusedDirectMessages"
-   - "ExperimentalChannelOrganization"
-   - "ExperimentalChannelSidebarOrganization"  
-
-10. Changes to ``mattermost-server/model`` for naming consistency.
+8. All legacy channel sidebar experimental configuration settings. We encourage customers using these settings to upgrade to v5.32 or later to access [custom, collapsible channel categories](https://mattermost.com/blog/custom-collapsible-channel-categories/) among many other channel organization features. The settings being deprecated include:
+   - ["EnableLegacySidebar"](https://docs.mattermost.com/administration/config-settings.html#enable-legacy-sidebar)
+   - ["ExperimentalTownSquareIsReadOnly"](https://docs.mattermost.com/administration/config-settings.html#town-square-is-read-only-experimental)
+   - ["ExperimentalHideTownSquareinLHS"](https://docs.mattermost.com/administration/config-settings.html#town-square-is-hidden-in-left-hand-sidebar-experimental)
+   - ["EnableXToLeaveChannelsFromLHS"](https://docs.mattermost.com/administration/config-settings.html#enable-x-to-leave-channels-from-left-hand-sidebar-experimental)
+   - ["CloseUnusedDirectMessages"](https://docs.mattermost.com/administration/config-settings.html#autoclose-direct-messages-in-sidebar-experimental)
+   - ["ExperimentalChannelOrganization"](https://docs.mattermost.com/administration/config-settings.html#sidebar-organization)
+   - ["ExperimentalChannelSidebarOrganization"](https://docs.mattermost.com/administration/config-settings.html#experimental-sidebar-features)
+
+9. [All configuration settings previously marked as “Deprecated”](https://docs.mattermost.com/administration/config-settings.html#deprecated-configuration-settings).
+
+10. Plugin API Changes:
+   - Remove deprecated ``model.CommandArgs.Session``.
+   - Restrict ``UserWillLogIn``, ``FileWillBeUploaded``, and ``MessageWillBe[Posted|Updated]`` hooks to enterprise licenced instances only.
+   - Changes to ``mattermost-server/model`` for naming consistency.
```

Comment:
These changes are not decided yet and hence we should not announce them. There is a meeting next week where they will be discussed and decided. The changes also slipped into https://mattermost.com/blog/looking-forward-to-mattermost-v6-0/.